src/Illuminate/Collections/Enumerable.php

```diff
@@ -338,11 +338,11 @@ public function whenNotEmpty(callable $callback, callable $default = null);
      * @template TUnlessReturnType
      *
      * @param  bool  $value
-     * @param  (callable($this): TUnlessReturnType)  $callback
+     * @param  (callable($this): TUnlessReturnType)|null  $callback
      * @param  (callable($this): TUnlessReturnType)|null  $default
      * @return $this|TUnlessReturnType
      */
-    public function unless($value, callable $callback, callable $default = null);
+    public function unless($value, callable $callback = null, callable $default = null);
```

Comment:
I think we can insert Nullable types, it was inserted in php 7.1 and Laravel 10 requires php 8.1. We can also insert this in Laravel 11.


```suggestion
    public function unless($value, ?callable $callback = null, ?callable $default = null);
```